projects/lib/src/configurator/color-picker/color-picker.component.ts

```diff
@@ -0,0 +1,98 @@
+import { Component, Input, OnChanges, SimpleChanges } from "@angular/core";
+import { FormsModule } from "@angular/forms";
+
+import { ConfiguratorContext } from "../configurator.models";
+import { NgModelChangeDebouncedDirective, TooltipDirective } from "../../utils";
+import { CommonModule } from "@angular/common";
+
+@Component({
+  selector: 'uib-color-picker',
+  imports: [CommonModule, FormsModule, TooltipDirective, NgModelChangeDebouncedDirective],
+  standalone: true,
+  template: `
+  <label for="color-{{property}}" class="form-label" *ngIf="label">{{label}}</label>
+  <div class="color-picker-wrapper mb-2">
+    <input [uib-tooltip]="tooltip" i18n-uib-tooltip i18n
```

Comment:
Here uib-tooltip is a variable, so the `i18n-uib-tooltip` attribute is not going to work.
But It would be useful on the button "Reset color" below.